src/components/books.jsx

```diff
@@ -7,19 +7,45 @@ function Book(props) {
   } = props;
 
   return (
-    <div>
-      <h3>{title || 'No Title'}</h3>
-      <p>
-        Author:
-        {' '}
-        {author || 'Unknown'}
-      </p>
-      <p>
-        Category:
-        {' '}
-        {category || 'Unknown'}
-      </p>
-      <button type="button" onClick={handleDelete}>Delete</button>
+    <div className="bookhead">
+      <div className="book-info">
+
+        <p>
+          {' '}
+          {category || 'Unknown'}
+        </p>
+        <h3>{title || 'No Title'}</h3>
+        <p>
+          {' '}
+          {author || 'Unknown'}
+        </p>
+        <div className="booksbuttons">
+          <button type="button">Comments</button>
+          <hr />
+          <button type="button" onClick={handleDelete}>Remove</button>
+          <hr />
+          <button type="button">Edit</button>
+
+        </div>
+      </div>
+      <div className="book-container">
+        <div className="oval-2" />
+      </div>
+      <div className="percentage">
+        <p>64%</p>
+        <p>completed</p>
+      </div>
+      <hr className="book-hr" />
+      <div className="book-last">
+        <p>
+          CURRENT CHAPTER
+
+        </p>
+        <p>
+          Introduction
+        </p>
```

Comment:
- [ ] Please note that the `font-family` for this element is `'Roboto Slab'`. Kindly implement this so that your design looks very similar to that of Zeplin's template. 👍 😎